lib/chef/knife/vra_server_create.rb

```diff
@@ -0,0 +1,134 @@
+#
+# Author:: Chef Partner Engineering (<partnereng@chef.io>)
+# Copyright:: Copyright (c) 2015 Chef Software, Inc.
+# License:: Apache License, Version 2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+require 'chef/knife'
+require 'chef/knife/cloud/server/create_command'
+require 'chef/knife/cloud/server/create_options'
+require 'chef/knife/cloud/vra_service'
+require 'chef/knife/cloud/vra_service_helpers'
+require 'chef/knife/cloud/vra_service_options'
+
+class Chef
+  class Knife
+    class Cloud
+      # rubocop:disable Style/BlockDelimiters
+      class VraServerCreate < ServerCreateCommand
+        include VraServiceHelpers
+        include VraServiceOptions
+        include ServerCreateOptions
+
+        banner 'knife vra server create CATALOG_ID (options)'
+
+        option :cpus,
```

Comment:
I chose to eliminate them because knife-cloud already includes SO MANY OPTIONS and coming up with unique short flags that don't collide with something else becomes quite difficult.  If users complain, I'll cave, but I think I'd like to leave as-is for now.